### PR TITLE
Added missing VB SimplifyInterpolationTests

### PR DESCRIPTION
**Pull request by jnm2**
@CyrusNajmabadi

### CODE REVIEW

File: src/EditorFeatures/VisualBasicTest/SimplifyInterpolation/SimplifyInterpolationTests.vb [hunk 1]
```diff
@@ -0,0 +1,611 @@
+﻿Imports Microsoft.CodeAnalysis.CodeFixes
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/VisualBasicTest/SimplifyInterpolation/SimplifyInterpolationTests.vb]: copyright.

**Comment by CyrusNajmabadi** [comment]
Thanks!

**Comment by CyrusNajmabadi** [comment]
@jnm2 tests are failing.  do you want to try to patch up?

**Comment by jnm2** [comment]
@CyrusNajmabadi Yes, on it.

**Comment by jnm2** [comment]
@CyrusNajmabadi There are two failing tests left and it appears to be a bug in SimplifyInterpolation/Helpers.cs. The string literal argument to `ToString` does not have a constant value according to IOperation even though the syntax is LiteralExpressionSyntax `"g"`. Still confirming.

**Comment by jnm2** [comment]
@CyrusNajmabadi Updated, looked carefully at each test this time.

**Comment by CyrusNajmabadi** [comment]
Thanks!

**Comment by jnm2** [comment]
@CyrusNajmabadi restart CI?

**Comment by jnm2** [comment]
Moving to the front of a new PR that adds more tests.

**Comment by jnm2** [comment]
Moved to https://github.com/dotnet/roslyn/pull/42681.

**Pull request closed without merging**